yt_dlp/extractor/livehockey.py

```diff
@@ -0,0 +1,33 @@
+from .common import InfoExtractor
+from .jwplatform import JWPlatformIE
+
+
+class LiveHockeyIE(InfoExtractor):
+    _VALID_URL = r'https?://livehockey\.com\.au/(?:[^/]+/)*(?P<id>[^/?#&]+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://livehockey\.com\.au/(?:[^/?#]+/)*(?P<id>[^/?#]+)'
```
Maybe this should be further refined, though, to only match pages with videos? Since the `only_matching` test does not have any jwplatform id to be extracted

Reply:
At this website it's not really possible to tell from the URL whether there is a video or not.